Synergism.css

```diff
@@ -20,6 +20,10 @@ html {
     scrollbar-width: none;
 }
 
+img {
+    image-rendering: pixelated;
```

Comment:
why is this needed? can you show screenshots of before/after?

Reply:
This is based on the forum thread found within suggestions on the discord server, it enforces the pixelated rendering style so that the pixel art of the game does not get blurred together via anti-aliasing.
https://discord.com/channels/677271830838640680/1112301231457706064
Before and After pictures are within the thread.

Reply:
thank you, it's fine - I just wanted to know what it was doing since it was being applied to all images.